howso/utilities/internals.py

```diff
@@ -563,12 +564,6 @@ def sanitize_for_json(obj: Any):  # noqa: C901
     # Serialize objects to dict
     if isinstance(obj, Mapping):
         obj_dict = obj
-    elif hasattr(obj, 'openapi_types') and hasattr(obj, 'attribute_map'):
```

Comment:
Might be good to keep this around, in case the user had some openapi type in their data somehow.

Reply:
Addressed by:  c28e6de8dded39f04ba3206735fe885f931d8d2e